catalog/app/containers/Bucket/PackageTree.js

```diff
@@ -178,7 +178,7 @@ function ExposeLinkedData({ bucketCfg, bucket, name, revision }) {
 function PkgCode({ data, bucket, name, revision, path }) {
   const code = data.case({
     Ok: ({ hash }) => {
-      const nameWithPath = path ? `${name}/${path}` : name
+      const nameWithPath = JSON.stringify(s3paths.ensureNoSlash(`${name}/${path}`))
```

Comment:
`ensureNoTrailingSlash`, I suppose

Reply:
Maybe `encodeURIComponent` is more appropriate than `JSON.stringify`?

Reply:
> ensureNoTrailingSlash

it would be more precise, but the function is called just `ensureNoSlash`

> Maybe encodeURIComponent is more appropriate than JSON.stringify?

we need simple special char escaping here to use the escaped strings in python and shell, not uri encoding